diagnostic.md

```diff
@@ -6,21 +6,22 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+      The Application Router specifies the URL for every viewstate of your app.
+      An Ember Route loads the model associated with a given route
```

Comment:
Yes, the Ember Application Router defines the route, and the Ember Route specifies how to load model data using the model hook.